src/v/config/configuration.cc

```diff
@@ -841,6 +841,13 @@ configuration::configuration()
       "Maximum size of a single request processed via Kafka API",
       {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
       100_MiB)
+  , kafka_batch_max_bytes(
+      *this,
+      "kafka_batch_max_bytes",
+      "Maximum size of a batch processed by server. If batch is compressed the "
+      "limit applies to compressed batch size",
+      {.needs_restart = needs_restart::no, .visibility = visibility::tunable},
+      1_MiB)
```

Comment:
should this be off by default (and on in freshly created clusters) in the next release, or the limit made large? the concern would be that an upgrade causes clusters to start rejecting workloads that were working before this. maybe sync with Roko/CS?

Reply:
good point, @mattschumpert do you have an opinion on that

Reply:
what was the conclusion here?

Reply:
I am not sure what the conclusion was but FTR the default limit was set to and remains at 1 MB today.